the-blue-alliance-ios/AppDelegate.swift

```diff
@@ -158,6 +164,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         navigationBarAppearance.titleTextAttributes = [NSAttributedStringKey.foregroundColor: UIColor.white]
     }
 
+    private func launchScreen() -> UIViewController {
```

Comment:
We probably don't want to kill the launch screen storyboard - I wonder if we can test this?